manifests/d/DuckStudio/ChineseGit/2.10.0/DuckStudio.ChineseGit.installer.yaml

```diff
@@ -1,22 +1,19 @@
 # Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
 
 PackageIdentifier: DuckStudio.ChineseGit
-PackageVersion: "2.10"
+PackageVersion: "2.10.0"
 InstallerLocale: zh-CN
 InstallerType: inno
 Scope: machine
 InstallModes:
 - interactive
 - silent
 - silentWithProgress
-InstallerSwitches:
-  Silent: /verysilent /suppressmsgboxes
-  SilentWithProgress: /verysilent
-UpgradeBehavior: install
 Dependencies:
   PackageDependencies:
   - PackageIdentifier: Git.Git
```

Comment:
Because of this dependency, this package may **never** pass the verification pipeline, please verify this package manually.

More:
- https://github.com/microsoft/winget-pkgs/pull/160675#issuecomment-2203586801 (version 2.7)
- https://github.com/microsoft/winget-pkgs/pull/163956#issuecomment-2241165257 (version 2.8)
- https://github.com/microsoft/winget-pkgs/pull/164755#pullrequestreview-2202191415 (version 2.9)
- https://github.com/microsoft/winget-pkgs/pull/164102#issuecomment-2248189203